test/unit/kubernetes-deploy/kubeclient_builder_test.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+require 'test_helper'
+require 'kubernetes-deploy/kubeclient_builder'
+
+class KubeClientBuilder < KubernetesDeploy::TestCase
```

Comment:
This class name should end in `Test` by convention

Reply:
Updated class name.